.travis.yml

```diff
@@ -0,0 +1,35 @@
+notifications:
+  email: false
+
+branches:
+  only:
+    - master
+    - develop
+    - /^v\d+\.\d+\.\d+/
```

Comment:
Whoa! What does this regex do?

Reply:
This basically checks for Git tags. So, if we push `v1.2.3`, Travis will know that it should run a build. We don't really maintain a `develop` branch, so we could likely get rid of that one, but I like to have it around in case the workflow style changes to Git flow at some point.

Reply:
Dope! That's so cool! Sounds good to me having `develop` in there, future-proofing for the win!